### ISSUE TITLE
ncurc with "pre" option broken since v12.5.10

### ISSUE THREAD

**Issue by ath0mas** [raineorshine/npm-check-updates#1103]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 12`

node v18.0.0, npm 8.6.0, Windows 10, ncu v2.5.9 or less, and v2.5.10

---------------------------

Please, since v12.5.10, `pre` option inside `.ncurc.json` results in `error: option '--pre <n>' argument missing` or `No dependencies.`.

## Steps to Reproduce
Blank dir,

then
```shell
$ npm install -E tslint@5.0.0
```
and `.ncurc.json`
```json
{
  "target": "latest",
  "pre": 1,
}
```

## Current Behavior
With _ncu_ v2.5.10 **and** `.ncurc.json` containing `pre` option:

```shell
$ ncu
error: option '--pre <n>' argument missing
```
```shell
$ ncu --target newest
Using config file C:\ncuFoo\ncuBar\.ncurc.json
Checking C:\ncuFoo\ncuBar\package.json

No dependencies.
```
```shell
$ ncu --pre 1
Using config file C:\ncuFoo\ncuBar\.ncurc.json
Checking C:\ncuFoo\ncuBar\package.json

No dependencies.
```

## Expected Behavior
With _ncu_ v2.5.10 **and** `.ncurc.json` NOT containing `pre` option,
or with _ncu_ v2.5.10 **and** NO `.ncurc.json`,
or with _ncu_ up to v2.5.9 in all those cases:

```shell
$ ncu
Using config file C:\ncuFoo\ncuBar\.ncurc.json
Checking C:\ncuFoo\ncuBar\package.json
[====================] 1/1 100%

 tslint  5.0.0  →  5.20.1
( tslint  5.0.0  →  6.0.0-beta1 -- depending)
```
```shell
$ ncu --target newest
Using config file C:\ncuFoo\ncuBar\.ncurc.json
Checking C:\ncuFoo\ncuBar\package.json
[====================] 1/1 100%

 tslint  5.0.0  →  6.1.3

Run ncu -u to upgrade package.json
```
```shell
$ ncu --pre 1
Using config file C:\ncuFoo\ncuBar\.ncurc.json
Checking C:\ncuFoo\ncuBar\package.json
[====================] 1/1 100%

 tslint  5.0.0  →  6.0.0-beta1

Run ncu -u to upgrade package.json
```

**Comment by raineorshine** [comment]
Thanks for reporting!

Fixed and published in `v12.5.11`

**Comment by ath0mas** [comment]
Thanks! I confirm it works great with v12.5.11.
